src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -179,9 +179,9 @@ def default_api_version(self):
         ResourceType.MGMT_RESOURCE_PRIVATELINKS: '2020-05-01',
         ResourceType.MGMT_NETWORK_DNS: '2018-05-01',
         ResourceType.MGMT_KEYVAULT: '2022-07-01',
-        ResourceType.MGMT_AUTHORIZATION: SDKProfile('2020-04-01-preview', {
+        ResourceType.MGMT_AUTHORIZATION: SDKProfile('2022-04-01', {
             'classic_administrators': '2015-06-01',
-            'role_definitions': '2018-01-01-preview',
+            'role_definitions': '2022-04-01',
```

Comment:
```suggestion
```
We can simply remove this line if `role_definitions` uses the same API version with default one~

Reply:
`role_definitions` frequently diverges from `role_assignments`, so it is merely a placeholder.

Reply:
`role_definitions` will again diverge from `role_assignments`: https://github.com/Azure/azure-cli/pull/26577